src/components/WebexMember/WebexMember.scss

```diff
@@ -34,6 +34,10 @@
     margin-right: 2.5rem;
   }
 
+  .unmuted {
+    fill: $wxc-green-40;
```

Comment:
this should come from a css variable: `--wxc-unmuted-color` (add it to the top just after `--wxc-text-color`)